pkg/imp/config/config.go

```diff
@@ -0,0 +1,48 @@
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/BurntSushi/toml"
+	"github.com/naggie/dstask"
+	"gopkg.in/yaml.v2"
+)
+
+type Config struct {
+	Github []Github
+}
+
+type Github struct {
+	Token        string
+	Repos        []string
+	GetClosed    bool `toml:"get_closed"`
+	Assignee     string
+	Milestone    string
+	Labels       []string
+	TemplateStr  string      `toml:"template_str"`
+	TemplateTask dstask.Task `toml:"-"`
+}
+
+func Load(configFile, repo string) (Config, error) {
+	var config Config
+
+	contents, err := ioutil.ReadFile(configFile)
+	if err != nil {
+		return config, fmt.Errorf("Couldn't read config file %q: %s", configFile, err.Error())
+	}
+
+	_, err = toml.Decode(string(contents), &config)
+	if err != nil {
+		return config, fmt.Errorf("Invalid config file %q: %s", configFile, err.Error())
```

Comment:
Is %q the correct template formatter here? I thought that was for integers
https://golang.org/pkg/fmt/#hdr-Printing

Reply:
It adds quotes around strings and escapes white space if necessary.  I like %q because it makes it more clear what exactly the given variable is (no confusion around white space )